DataBaseGUI.java

```diff
@@ -24,60 +8,58 @@
 	/**
 	 * 
 	 */
-	private static final long serialVersionUID = -5292619009088841443L;
+	private static final long serialVersionUID = 1L;
 	public static String s = "";
 	ArrayList<Student> list = new ArrayList<>();
-	private JButton OK, CANCEL, SHOWALL, CLEARHISTORY, SAVE, SAVEALL; 	
-	private JTextField FieldName, FieldAge;
-	private JLabel LabelName, LabelAge, LabelCombo;
-	private JComboBox<Integer> Combo;
+	private JButton ok, cancel, showAll, clearHistory, save, saveAll; 	
+	private JTextField fieldName, fieldAge;
+	private JLabel labelName, labelAge, labelCombo;
+	private JComboBox<Integer> combo;
+	public final static Integer[] yearsDataSource = {1, 2, 3, 4, 5};
 	
 	public DataBaseGUI (){
 	super ("Student Database");
 	setBounds(400, 380, 450, 100);
 	setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
 	setResizable(false);
 	
-	OK = new JButton("OK");
-	CANCEL = new JButton("Cancle");
-	SHOWALL = new JButton("Show history");
-	CLEARHISTORY = new JButton("Clear history");
-	SAVE = new JButton("SAVE");
-	SAVEALL = new JButton("Save all");
+	ok = new JButton("OK");
```

Comment:
Include object type in variable name. This will significantly increase code readability